### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by abdulrahmanshr75**
Kindly review my changes.
thank you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,11 @@
...
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
```
Comment (VanessaAoki) [inline README.md:72]: - [ ] Great job using Microverse's README template, but this section, that was deleted, should be in the file and edited with your own information.

File: README.md [hunk 2]
```diff
@@ -1,75 +1,11 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> One paragraph statement about the project.
+Hello Microverse
 
 ![screenshot](./app_screenshot.png)
```
Comment (VanessaAoki) [inline README.md:3]: - [ ] You should consider adding one paragraph describing your project and what it is about.
- [ ] You should also consider adding a screenshot of your own project.

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @abdulrahman75, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Correct use of the GitHub Flow :fire: 

## Required Changes ♻️
_Check the comments under the review._
#### Besides those changes, you should also consider:
- [x] You should have meaningful commit messages ("update linters" is good, but "first commit" or "commit" is not).
- [ ] Your pull Request should have a short summary describing introduced changes in more details.

so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alonsofl — APPROVED** [review]
Hi @abdulrahmanshr75,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**